### PR TITLE
Add Keck atmosphere from KAON303

### PR DESCRIPTION
**Pull request by vkooten**
this adds the feature from issue #151

### CODE REVIEW

File: hcipy/atmosphere/standard_atmosphere.py [hunk 1]
```diff
@@ -4,54 +4,81 @@
...
+    ----------
+    input_grid : Grid
+
+    Returns
+    -------
+    list
+        A list of turbulence layers.
+    '''
+
+    heights = np.array([0.0, 2.1, 4.1, 6.5, 9.0, 12.0, 14.8]) * 1000
```
Comment (syhaffert) [inline hcipy/atmosphere/standard_atmosphere.py:75]: I would write out the layer heights in meters. We have done this too for the other models.

**Comment by syhaffert** [comment]
I think something happened with the white spaces. Are you using the same indentation style as hcipy?

File: hcipy/atmosphere/standard_atmosphere.py [hunk 2]
```diff
@@ -55,3 +55,30 @@ def make_las_campanas_atmospheric_layers(input_grid, r0=0.16, L0=25, wavelength=
 		layers.append(InfiniteAtmosphericLayer(input_grid, cn, L0, v, h, 2))
 
 	return layers
+
+def make_keck_atmospheric_layers(input_grid):
+
```
Comment (syhaffert) [inline hcipy/atmosphere/standard_atmosphere.py:60]: Remove line 60. There are no empty lines after the function definition.

**Review by syhaffert — COMMENTED** [review]
This is good to merge after you made the small formatting changes.

**Comment by ehpor** [comment]
@vkooten Taking this over to push this over the finish line.

**Review by ehpor — APPROVED** [review]
Technically correct. The atmospheric layers in general require more work so this is good enough for now.

**Pull request merged**